test/jest/victory-core/victory-label/victory-label.test.js

```diff
@@ -0,0 +1,246 @@
+import React from "react";
+import { VictoryLabel, Log } from "victory-core";
+import { screen, fireEvent } from "@testing-library/react";
+import { renderInSvg } from "../../rendering-utils";
+
+describe("components/victory-label", () => {
+  it("accepts user props", () => {
+    renderInSvg(
+      <VictoryLabel
+        data-testid="victory-label"
+        aria-label="test-aria-label"
+        text="label"
+      />
+    );
+
+    expect(screen.getByTestId("victory-label")).toBeDefined();
+    expect(screen.getByLabelText("test-aria-label")).toBeDefined();
+  });
+
+  it("has expected content with render", () => {
+    const { container } = renderInSvg(<VictoryLabel text="such text, wow" />);
+    expect(container.querySelector("tspan").innerHTML).toMatchInlineSnapshot(
+      `"such text, wow"`
+    );
+  });
+
+  it("sets dx and dy for text element", () => {
+    const { container } = renderInSvg(
+      <VictoryLabel dx={30} dy={30} text="such text, wow" />
+    );
+    const output = container.querySelector("text");
+    expect(output.getAttribute("dx")).toEqual("30");
+    // dy = props.dy + (capHeight(0.71) / 2 + (0.5 - length(1) / 2) * lineHeight(1)) * fontSize(14);
+    expect(output.getAttribute("dy")).toBeNull();
```

Comment:
I'm not sure why this is null when dy is one of the attributes being tested? But I'm guessing it was an intentional change at some point.